python/packages/autogen-core/src/autogen_core/application/_worker_runtime.py

```diff
@@ -748,3 +756,18 @@ async def get(
 
     def add_message_serializer(self, serializer: MessageSerializer[Any] | Sequence[MessageSerializer[Any]]) -> None:
         self._serialization_registry.add_serializer(serializer)
+
+    async def _process_cloud_event(self, cloud_event: cloudevent_pb2.CloudEvent) -> None:
+        logger.info(f"Processing CloudEvent: {cloud_event}")
+
+        event = agent_worker_pb2.Event(
+            topic_type=cloud_event.type,
+            topic_source=cloud_event.source,
+            payload=agent_worker_pb2.Payload(
+                data_type=cloud_event.type,
+                data=cloud_event.proto_data.value,
+                data_content_type=JSON_DATA_CONTENT_TYPE,
+            ),
+            metadata=get_telemetry_grpc_metadata(),
+        )
+        await self._process_event(event)
```

Comment:
Just leave a note here. We need to wrap RpcRequest, RpcResponse and Event in to CloudEvent.